docs/config.md

```diff
@@ -100,6 +100,7 @@ Currently the tool has a rather naive implementation for mapping certain constru
 |MapArray|Maps an array by replacing comma with semi-colon|
 |MapRemainingWork|Maps and converts a Jira time to hours|
 |MapRendered|Maps field to rendered html format value|
+|MapLexoRank|Maps and converts a Jira LexoRank to decimal. When mapping this type of field, ensure the correct Jira custom field is used and mapped to the relevant Azure DevOps prioritization field (see: https://learn.microsoft.com/en-us/azure/devops/boards/queries/planning-ranking-priorities?view=azure-devops#fields-used-to-plan-and-prioritize-work)|
 |(default)|Simply copies soure to target|
 
 ## Example configuration
```

Comment:
Perhaps we can do a little note here that this example applies to the Scrum template? Agile and CMMI should use StackRank.

Reply:
Can’t see the rest of the file after this comment, but doesn’t the config specify “scrum” as the process? Maybe add an alternative example that is not scrum to show this?